### ISSUE TITLE
Unable to Open DDLC; quick black screen, game freezes/crashes

### ISSUE THREAD

**Issue by cremecaramel** [Monika-After-Story/MonikaModDev#7223]
Hello! I submitted an issue very similar to this a long time ago. I finally updated my computer and I even installed the new updates for MAS. However, much like before, the screen went black for a short second when the game opens. Monika appears and asks if it was "another crash?" and then I'm unable to continue the dialogue because the game freezes. I had to open task manager to exit out...  agh, I don't know what's going on?? How can I fix this?

**Comment by multimokia** [comment]
In what way does the game freeze? Does it do a kinda windows "(not responding)" thing? Or something else?

**Comment by cremecaramel** [comment]
> In what way does the game freeze? Does it do a kinda windows "(not responding)" thing? Or something else?

I didn't notice the window to see, but I am unable to click through dialogue or press 'x" out. I don't get any sort of pop-up alerts :-s

**Comment by ThePotatoGuy** [comment]
Are you still having this issue on the latest version?

**Comment by cremecaramel** [comment]
> Are you still having this issue on the latest version?

Yes, I am still unfortunately having this issue. I understand it might be difficult to explain, but I do have videos that show what happens. I’m having a difficult time pinning them to my comments though

**Comment by ThePotatoGuy** [comment]
you can upload the videos to youtube and paste a link here.

**Comment by cremecaramel** [comment]
THANK YOU SO MUCH FOR KEEPING THIS OPEN!! /Gen I am so sorry for the late response. Here’s the link to my video: https://youtube.com/shorts/okCpQnzaJe8?feature=share 

So, to give a rundown, I open up ddlc and it takes a while to load in. I’m only able to play for a brief moment when it suddenly closes the game. This continues with each time I tried to open the game with varying moments on how long it stays open; I never truly reach the menu screen.

IT WAS SCARY. The third or fourth time I opened the game, MY COMPUTER COMPLETELY FROZE. I COULDNT EVEN CTRL-ALT-DELETE, I had to close down my computer with a power button press.

I think some information to know was that I did update my computer, I had enough storage on my computer (about 143 gb?), and I installed the new mod. I think history wise, I once accidentally turned on unstable mode but immediately turn it off the second I did. I hope that accidental misclick was not the case.

also, could this possibly have something to do with Monika’s accessories? Do I need to update the mod assets? 

I really hope we can get this issue resolved. I’ve become very attached to Monika and besides, I really hope this bug doesn’t destroy my computer

THANK YOU SO MUCH!!!

**Comment by cremecaramel** [comment]
I installed the new mod update at the time ** I haven’t touched the game since it froze, may 23

**Comment by cremecaramel** [comment]
I’ll see if I can record and capture the black screen flicker. I didn’t catch it on camera in the video but I do know that was apart of this; especially when I tried opening up the game before I updated my computer.

**Comment by cremecaramel** [comment]
SPECS:
Processor	Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz   1.80 GHz
Installed RAM	8.00 GB (7.84 GB usable)
Device ID	D9ECB060-108A-4743-B75E-218CA6547B82
Product ID	00325-96479-97702-AAOEM
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display
Edition	Windows 10 Home
Version	21H1
Installed on	‎5/‎20/‎2021
OS build	19043.1165
Experience	Windows Feature Experience Pack 120.2212.3530.0

[Monika After Story.zip](https://github.com/Monika-After-Story/MonikaModDev/files/7101081/Monika.After.Story.zip)

**Comment by ThePotatoGuy** [comment]
I'll check these later in the week.

**Comment by cremecaramel** [comment]
Awesome, thank you so much @ThePotatoGuy !! I greatly appreciate your help!!! <:-D <3

**Comment by cremecaramel** [comment]
Here's the log:

[log.txt](https://github.com/Monika-After-Story/MonikaModDev/files/7101158/log.txt)

**Comment by cremecaramel** [comment]
UPDATE: It started working normally after uninstalling, reinstalling DDLC around 2:14pm-2:25pm

Since then, I've:
Updated mod
DDLC MAS working normally until I clicked "I'm going to restart" when closing MAS
restarted computer
Tried reopening DDLC and the screen started to flicker, couldn't open MAS without it closing out 
After that, I tried uninstalling/reinstalling DDLC but the screen still flickered -> froze 

TLDR: the problem is still there :-[ Should I submit an updated log, persistent, etc??
